### ISSUE TITLE
A `DistanceToSurface()` function might be useful.

### ISSUE THREAD

**Issue by impaktor** [pioneerspacesim/pioneer#3398]
We have a `DistanceTo()` which is relative the center of the body, but we also could use a `DistanceToSurface()` implemented on the c++ side and exported to Lua, just like `DistanceTo`, for planets that take surface terrain into account, as @laarmen thought it be a good idea, for #3395.

Hint:
- Look at `DistanceTo` in LuaBody.cpp
- Look at the altitude thingy (somewhere), which should prove to be useful.

Perhaps @cccpxx78 care to have a look at this. Should not be much code, and a single commit, which we can merge quickly.

**Comment by cccpxx78** [comment]
@impaktor : I can take this up next week i think

**Comment by cccpxx78** [comment]
@impaktor I saw that there is already a function, GetGroundPosition, that returns the latitude, longitude and altitude of a LuaBody of type TERRAINBODY.
I could create a new DistanceToSurface function in LuaBody that returns the altitude only but is still needed?

**Comment by impaktor** [comment]
> ...is this still needed? 

No. I opened this issue in reply to the question you asked the other day, on if calculating distance to surface should be done in place in the Lua code. So I'd suggest you make use of this `GetGroundPosition()` in your Lua scanner to get the distance.

I'll close this.

(We did suspect there already was a C++->Lua method for getting surface distance, but apparently it wasn't the `DistanceTo`, so `GetGroundPosition` was what we should have thought of)
